scripts/install

```diff
@@ -62,11 +62,13 @@ create_unix_user
 
 # Prepare venv
 init_virtualenv
+sudo /opt/yunohost/ihatemoney/venv/bin/pip install 'pip<10'
```

Comment:
Why pip<10 ?

(nitpicking : push it to the next line ;-)

Reply:
When trying to install ihatemoney package with pip, I get the error "ImportError: No module named 'pip.req'". It appears this error comes with pip10 changes in the api.

Reply:
Oh yeah, I remember now…That is linked to https://github.com/spiral-project/ihatemoney/issues/340

In recent versions of Ihatemoney, this is fixed, but we are still on an old version.

IHM update itself is for another PR another time.

Well spotted :-)